.github/workflows/integration.yml

```diff
@@ -202,6 +206,15 @@ jobs:
           max_attempts: 3
           command: bash scripts/ci-integration-test.sh  datadog-traces
 
+      - if: (github.event_name == 'merge_group' || needs.changes.outputs.all-int == 'true' || needs.changes.outputs.datadog-e2e-logs == 'true') &&
+          (github.event_name != 'pull_request' || env.PR_HAS_ACCESS_TO_SECRETS == 'true')
+        name: datadog-e2e-logs
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 60
```

Comment:
Note: I'm trying this out because the 30 min timed out when this was the only one that needs to be exercised.

Since most of the time we'd be using the shared image, this wouldn't be an issue (eg merge queue).

However, we could consider pulling these e2e tests out into a separate workflow that runs nightly or weekly.

Reply:
It took 45 mins. Extra time because there is more stuff being built.

Reply:
Is this still applicable given that we removed the redundant build step?

Reply:
good catch. It actually too 29 mins ... that is cutting it a little close for comfort so I'll set it to 35 min timeout.